src/main/java/io/syndesis/qe/springboot/http/controller/auth/AuthController.java

```diff
@@ -46,11 +55,20 @@ public String test(){
         return "Success";
     }
 
+    public static String encodeExpectedPasswordAndUsername() {
+        String auth = BASIC_USERNAME + ":" + BASIC_PASSWORD;
+        byte[] encodedAuth = Base64.encodeBase64(auth.getBytes(StandardCharsets.US_ASCII));
```

Comment:
I'd personally go for UTF-8 just to be safe if we ever decide to use emoji passwords.